tests/accessor/accessor_exceptions_core.cpp

```diff
@@ -26,21 +26,21 @@ namespace accessor_exceptions_test_core {
 using namespace sycl_cts;
 
 DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
-("Generic sycl::accessor constructor exceptions test core types.",
+("Generic sycl::accessor constructor exceptions test. core types.",
```

Comment:
```suggestion
("Generic sycl::accessor constructor exceptions test. Core types.",
```

Reply:
I've applied this comment in his commit: https://github.com/KhronosGroup/SYCL-CTS/pull/359/commits/92f3e15e74282f88fceb1b5c98e796a214ef88a6